src/components/Common/video-card-flat.component.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Fragment } from 'react';
+import { Link } from 'react-router-dom';
+import formatter from '../../utils/formatter';
+
+const VideoCardFlat = (props) => {
+  const { snippet } = props;
```

Comment:
You can make this prop detached from the response structure by only passing the src, title, channelTitle, and publishTime instead of all the object.
```suggestion
  const { thumbnalURL, title, channelTitle, publishTime } = props;
```